src/lib/auth/account.ts

```diff
@@ -65,16 +98,19 @@ const initializeFilesystem = async (fs: FileSystem): Promise<void> => {
   await fs.mkdir(webnative.path.directory(...ACCOUNT_SETTINGS_DIR))
 }
 
-export const loadAccount = async (username: string): Promise<void> => {
+export const loadAccount = async (hashedUsername: string, fullUsername: string): Promise<void> => {
   const session = await getStore(sessionStore).authStrategy.session()
 
   filesystemStore.set(session.fs)
 
   const backupStatus = await getBackupStatus(session.fs)
 
+  localStorage.setItem(USERNAME_STORAGE_KEY, fullUsername)
```

Comment:
i'm saving the full username with the `#:<DID>` because it will be needed to generate the recovery kit

Reply:
I'd use `components.storage.setItem` instead, that way you're adding this to the namespace of your app. Probably best if you make the `program` available through a store (seeing that you need the crypto component as well).

Reply:
@icidasset ahh, didn't know about `components.storage.setItem`! that sounds like a much better approach, thanks :)

Reply:
updated!